CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- fix: upgrade Fluentd to 1.14.4-sumo-1 [#2057][#2057]
```

Comment:
I'm referring to this.

Commit says:

> fix: upgrade Fluentd to 1.14.4-sumo-1

but it's placed in `Changed` section

Reply:
I marked it as a "fix", because this upgrades is a security fix. But I can change it to `chore` as @swiatekm-sumo says.